sqrl/graphql/schema.py

```diff
@@ -118,6 +120,16 @@ def resolve_organisations(
         """Return a list of _OrganisationObject objects."""
         return list(Organisation.objects.all())
 
+    def resolve_courses_by_id(
+            self, info: graphene.ResolveInfo, ids: list[str]) -> list[Course]:
+        """Return a list of _CourseObject objects, each with given ids.
+        Courses that do not exist are ignored.
+        """
+        courses = []
+        for id in ids:
+            courses.append(Course.objects.get(id=id))
```

Comment:
won't this potentially add None to the list?

Reply:
I will take a closer look at this.

Reply:
This should be fixed, thanks for the heads-up.